ElectronClient/app/app.js

```diff
@@ -512,12 +512,20 @@ class Application extends BaseApplication {
 
 		function _showAbout() {
 			const p = packageInfo;
+			let gitInfo = '';
+			if ("git" in p) {
+				gitInfo = _('Revision') + ": " + p.git.hash + ' (' + p.git.branch + ')';
```

Comment:
If we're going to translate this, we need to use sprintf syntax: `_('Revision: %s (%s)', p.git.hash, p.git.branch)`. This is because different languages write this differently. For example, in French there's a space before the colon.

Reply:
ah, I see. thanks.